Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Less Likely Interracial AI

## Changelog:
- The AI is less likely to seduce, romance, and marry different race:
   - When you try to marry, you will get the Different Race debuff.
   - When you try to Seduce or Romance, the success chance will be lowered.
   - The AI never takes concubines of a different race.
   - The AI doesn't create interracial couples with and between its courtiers.
   - The human and high elven AI tolerates interracial marriages between each other by 50%.
   - The human and elven AI tolerates interracial marriages between each other by 25%.


# How to test:
Enter the observe mode and observe the game for a few years. Pay special attention to the Horde since it settles on human and dwarven lands.
